doc/usersguide/source/model.rst

```diff
@@ -219,6 +219,10 @@ However, in practice, there are often some fields which are not present in the
 input file. They will need to be calculated later by the model, and you need to
 tell LIAM2 that the field is missing, by using `initialdata: False` in the
 definition for that field (see the *agegroup* variable in the example below).
+The fields that are not present in the initial file can also be initialized to
+a specific value by using the `default: some_default_value`. If not present they
+are initialized to the default of the field type which are False for boolean and
+0 for integer and float (see the *alive* variable in the example below).

```

Comment:
You do no speak about new(). In fact I think a separate paragraph and saying it applies to both new() and fields not present in the input bla bla... would be better, what do you think?